homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -453,6 +456,23 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         parent_key=VacuumCoordinatorDataAttributes.last_clean_details,
         name="Last Clean Area",
         entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+    ),
+    f"current_{ATTR_STATUS_CLEAN_TIME}": XiaomiMiioSensorDescription(
+        native_unit_of_measurement=TIME_SECONDS,
```

Comment:
We don't want relative times in the state machine. In that case expose a timestamp from when the cleaning started.

Reply:
This sensor is supposed to expose for how long the current cleaning cycle has been running for. There is another sensor that exposes the start time of the cycle. 

So these are 2 separate sensors.

Reply:
The value of this sensor doesn't update just because time passes, right? It updates when the device has been used and the clean time increases. Is that correct?

Reply:
So:
t1: I press start on the vacuum
t2: vacuum has been cleaning for x1 seconds
t3: coordinator update is ran, sensor is set t x1 seconds
t4: vacuum has been cleaning for x2 seconds 
t5: coordinator update is ran, sensor is set to x2 seconds
etc 

So basically, indeed, it updates during the time spent cleaning during the current clean cycle. Once the cleaning cycle is finished, this sensor will stop updating. Until another clean cycle has started of course.

I did mention before that there is already a sensor indicating start time, but this sensor only updates once the cleaning cycle has finished as explained in https://github.com/home-assistant/core/issues/59272#issuecomment-986297000

I think I understand the feedback better now. So @balloob feedback is to just use the start time of the current cleaning cycle and then users can use templates etc to figure out for how long the cycle has been running?

Reply:
Since the sensor only updates when the device is actually used, I think it's ok to have a relative time as value, but it would be better to report in minutes. Seconds is too high resolution and most likely not interesting. We do allow formatting times before storing in state machine.

Reply:
Actually, this is then blocked on https://github.com/home-assistant/core/pull/59092 if we want to move an away from seconds. 

@rytilahti will you push that PR, so it gets merged, or do you want me to pick it up?

Reply:
Or this one gets merged as seconds and then picked up in the above linked PR.

Reply:
If we expect to break the value, we should close this PR and have it be included in the other PR ?

Reply:
That other PR is for merely for adjusting the sensor granularity / units, so this PR here should probably just be rebased on top of that, either pre or post merge.

Reply:
The PR to adjust the measurement unit was closed as the seconds as units was chosen to be the preferred format, so I suppose this PR can now proceed.

Reply:
Will have a look later today or tomorrow.